### PR TITLE
terminal hiding

### PR DESCRIPTION
**Pull request by zekth**
adress: https://github.com/qmk/qmk_configurator/issues/222

### CODE REVIEW

File: src/components/StatusPanel.vue [hunk 1]
```diff
@@ -25,7 +31,12 @@ export default {
     }
   },
   computed: {
-    ...mapGetters('status', ['message', 'scrollToLatest'])
+    ...mapGetters('status', ['message', 'scrollToLatest']),
+    isTerminalVisible: {
+      get() {
+        return this.$store.state.app.terminalVisible;
```
Comment (yanfali) [inline src/components/StatusPanel.vue]: Use mapState

File: src/jquery.js [hunk 2]
```diff
@@ -436,6 +436,14 @@ function enableCompileButton() {
   }, 2000);
 }
 
+function hideTerminal() {
+  store.commit('app/setTerminalVisibility', false);
```
Comment (yanfali) [inline src/jquery.js]: mapMutations

Comment (yanfali) [reply src/jquery.js]: These don't belong in here

**Comment by yanfali** [comment]
OK, I just tried it out. It looks funky. I think we need a 1 line height bar in black with some text that says something like "Expand Status". We can make it clickable and it will toggle the shrinking expanding, kind of like a concertina or unfolding of an editor. What do you think?

**Comment by zekth** [comment]
I can do this yes.

**Comment by zekth** [comment]
@yanfali do we still want this one? If so i'll rebase it.

**Comment by yanfali** [comment]
Closed in favor of #583

**Pull request closed without merging**